### DISCUSSION TITLE
Audio/Stream quality

### DISCUSSION THREAD

**Discussion by Nutti85** [music-assistant/support#1092] [Q&A] [answered]
I'm currently testing Qobuz with a WiiM Mini which comes up as a media player.  I'm wondering if Music Assistant touches the streaming quality in any way or if I'm getting the full experience with 24-bit 192 khz?

**Comment by marcelveldt** [comment, marked as answer]
Are you using it with DLNA ? In that case you should get the full original sound quality but make sure it is not re-encoded to some lossy format like MP3. Have a look at the Queue --> Settings to find out what audio format is sent to the player. I'd suggest using FLAC so it is lossless but a bit compressed to save some bandwidth.

**Reply by Nutti85** [reply]
Thanks for the tip. Qobuz uses FLAC and it seems like I get the highest quality

<img width="224" alt="image" src="https://user-images.githubusercontent.com/17089762/217095200-98b8f683-372c-475f-aeac-e1658d748ec8.png">
